config/routes.rb

```diff
@@ -1,8 +1,8 @@
 Rails.application.routes.draw do
-  resources :posts
-  resources :users
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root 'users#index'
+
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show]
+  end
```

Comment:
- You maintained RESTful URL, well done
-
![Capture](https://user-images.githubusercontent.com/63972670/185982943-aefd37c4-7390-4abc-857d-5c4df312ba72.JPG)

Reply:
Hi @OLIPLICHE, thank you for your review, I would like to inform you that I fixed all the optional issues you mentioned above.